bin/opteadm/build.rs

```diff
@@ -0,0 +1,31 @@
+use std::error::Error;
+use std::process::Command;
+
+fn main() -> Result<(), Box<dyn Error>> {
```

Comment:
No real need to change this, but I find `anyhow` excellent for these kinds of one-off, mostly string-based errors in small applications.